serverless/installing-openshift-serverless.adoc

```diff
@@ -12,8 +12,10 @@ include::modules/machineset-manually-scaling.adoc[leveloffset=+2]
 
 
 // Installing Service Mesh
-include::modules/serverless-installing-service-mesh-no-kiali-jaeger.adoc[leveloffset=+1]
+// Commented out for this release. May apply for future releases 
+// include::modules/serverless-installing-service-mesh-no-kiali-jaeger.adoc[leveloffset=+1]
 
+OpenShift Serverless requires an installed version of xref:../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[Service Mesh].
```

Comment:
This document points to the whole Service Mesh installation which includes installing ServiceMeshControlPlan and ServiceMeshMemberRoll. But we also include these steps in our documentation. 
I'd say it would be better to include just what is necessary from the Service Mesh docs and that is installing their operators: https://github.com/openshift/openshift-docs/blob/master/modules/ossm-operatorhub-install.adoc
And since we're deleting the file which had a note about creating the istio-system namespace and this step is required, I'd insert this step in https://github.com/openshift/openshift-docs/blob/master/modules/serverless-installing-service-mesh-control-plane.adoc just before the last step which applies the smcp.yaml